pkg/network/protocols/http/monitor.go

```diff
@@ -149,7 +162,12 @@ func (m *Monitor) Start() error {
 }
 
 func (m *Monitor) GetUSMStats() map[string]interface{} {
-	if m == nil {
+	if m == nil || state != Running {
```

Comment:
You could just return it all: 
```go
map[string]interface{}{
  "last_check": m.telemtry.then,
  "state": state,
  "error": startupError,
}
```

Reply:
can't do that like that, modified the suggestion